pkg/cmd/prune-backups/prune.go

```diff
@@ -33,10 +33,14 @@ type PruneOpts struct {
 	RetentionType      string
 	MaxNumberOfBackups int
 	MaxSizeOfBackupsGb int
+	BackupPath         string
```

Comment:
please remove all prune related changes from this PR

Reply:
done 👍🏽